app/src/components/Card.jsx

```diff
@@ -19,11 +19,11 @@ const Card = ({ props }) => {
             </span>
           )}
           {data.venue_name == "ONLINE" && (
-            <p className="info">Livestream, {data["time-begin"]}</p>
+            <p className="info">Livestream, {data.time_start}</p>
```

Comment:
data? is it artist? event? concert?

Reply:
This is not my function, im guessing it is data as in just the data reseved. Only changed to time_start.